### PR TITLE
updated access logs in destination for alb

### PR DESCRIPTION
**Pull request by Sowmyak442**

### CODE REVIEW

**Review by michael-fleming — CHANGES_REQUESTED** [review]
for the bucket name please make this a variable with a default value - we shouldn't have hardcoded values in the main file

**Comment by Sowmyak442** [comment]
> for the bucket name please make this a variable with a default value - we shouldn't have hardcoded values in the main file

updated

File: terraform/modules/loadbalancer/variables.tf [hunk 1]
```diff
@@ -27,7 +38,7 @@ variable "alb_internal" {
 
 variable "alb_ssl_policy" {
   description = "specify ssl policy to use"
-  default     = "ELBSecurityPolicy-TLS-1-2-Ext-2018-06"
+  default     = "ELBSecurityPolicy-TLS13-1-2-2021-06"
```
Comment (colemandevries) [inline terraform/modules/loadbalancer/variables.tf:41]: Thank you for this!

**Review by colemandevries — APPROVED** [review]
@Sowmyak442 I think that we should consider encouraging the hard-coded value in the main file for security-related configurations. In this context, we don't want to offer an opportunity for an engineer to provide an alternative ssl policy. 

Furthermore, we want to use some of our Terraform code as ATO evidence. If we have a hard-coded value, we can more confidently say all new ALBs will have the compliant ssl policy, and hopefully we can prove the same for the existing ALBs. 

Something to think about! For now, I think it's fine. Will approve.

**Comment by colemandevries** [comment]
Wanted to post a baseline configuration reference guide for everyone on this call, a request that was assigned by @akmukherjee . We can discuss in an upcoming DevOps call, but here is the link to the draft: 

https://nih.sharepoint.com/sites/NCI-CTOSDevOps/SitePages/Application-Load-Balancer.aspx

**Pull request merged**